spec/forms/collection_edit_form_spec.rb

```diff
@@ -19,4 +19,24 @@
       it { is_expected.to be true }
     end
   end
+
+  describe '#select_files' do
+    context 'without any works/files attached' do
+      subject { form.select_files }
+      it { is_expected.to be_empty }
+    end
+
+    context 'with a work/file attached' do
+      let(:work) { create(:work_with_one_file) }
```

Comment:
Good catch about persisting the collection @jcoyne. The collection needs to be persisted in order to for the #select_files method but not for the other specs.